t/debug/hook.t

```diff
@@ -127,3 +132,28 @@ hello world
 --- error_log
 filter(): call require("apisix.plugin").filter() args:{
 filter(): call require("apisix.plugin").filter() return:{
+
+
+=== TEST 5: missing hook_conf
+--- debug_config
+basic:
+  enable: true
+http_filter:
+  enable: true         # enable or disable this feature
+  enable_header_name: X-APISIX-Dynamic-Debug # the header name of dynamic enable
+
+hook_test:                      # module and function list, name: hook_test
+    apisix.plugin:              # required module name
+    - filter                    # function name
+
+#END
+--- request
+GET /hello
+--- more_headers
+Host: foo.com
+--- response_body
+hello world
+--- no_error_log
+[error]
```

Comment:
We can't check there is no `[error]` because `failed to validate debug config property "hook_conf" is required` is an error level log.